include/image.mk

```diff
@@ -591,18 +606,35 @@ endef
 
 define Device/DumpInfo
 Target-Profile: DEVICE_$(1)
-Target-Profile-Name: $(DEVICE_TITLE)
+Target-Profile-Name: $(DEVICE_DISPLAY)
 Target-Profile-Packages: $(DEVICE_PACKAGES)
 Target-Profile-hasImageMetadata: $(if $(foreach image,$(IMAGES),$(findstring append-metadata,$(IMAGE/$(image)))),1,0)
 Target-Profile-SupportedDevices: $(SUPPORTED_DEVICES)
 $(if $(DEFAULT),Target-Profile-Default: $(DEFAULT))
 Target-Profile-Description:
 $(DEVICE_DESCRIPTION)
+$(if $(strip $(DEVICE_ALT0_TITLE)),Alternative device titles:
+- $(DEVICE_ALT0_TITLE))
+$(if $(strip $(DEVICE_ALT1_TITLE)),- $(DEVICE_ALT1_TITLE))
+$(if $(strip $(DEVICE_ALT2_TITLE)),- $(DEVICE_ALT2_TITLE))
 @@
 
 endef
 
 define Device/Dump
+ifneq ($$(strip $$(DEVICE_ALT0_TITLE)),)
+DEVICE_DISPLAY = '$$(DEVICE_ALT0_TITLE)' ($$(DEVICE_TITLE)
```

Comment:
Missing closing bracket here.

Reply:
And can you explain the reason for the quotes (I'm curious :-))
(Quotes are inconsitent BTW: single here, double for the other two lines below)

Reply:
Single quotes are definitely too much
![Screenshot from 2019-08-20 10-46-48](https://user-images.githubusercontent.com/2287111/63327969-f30f6200-c337-11e9-8ff4-fd24a015e125.png)

Reply:
I just wondered why you use quotes at all here

Reply:
Yea sorry a leftover from testing to trick the Perl sorting. I just removed it and updated the commit message.